apps/zero-runtime-next-app/src/components/Avatar/Avatar.js

```diff
@@ -68,8 +69,8 @@ const AvatarRoot = styled('div', {
               backgroundColor: theme.vars.palette.Avatar.defaultBg,
             }
           : {
-              backgroundColor:
-                theme.palette.mode === 'light' ? theme.palette.grey[400] : theme.palette.grey[600],
+              backgroundColor: theme.palette.grey[400],
+              ...theme.applyDarkStyles({ backgroundColor: theme.palette.grey[600] }),
```

Comment:
You'll also need to remove `applyDarkStyles` override added in the `next.config.js` file.